config/rootfs/debos/overlays/cros-flash/opt/chromeos/cr50_updater

```diff
@@ -0,0 +1,127 @@
+#!/bin/sh
+# SPDX-License-Identifier: LGPL-2.1-or-later
+# Copyright (C) 2023 Collabora Ltd.
+# Author: Denys Fedoryshchenko <denys.f@collabora.com>
+
+if [ -z "$1" ]; then
+    echo "Error: no URL for firmware"
+    echo "Usage: $0 <URL>"
+    echo "Where <URL> is base URL for both prod and prepvt, e.g. https://storage..../dir"
+    echo "Both <URL>/cr50.bin.prod and cr50.bin.prepvt must be available"
+    exit 1
+fi
+
+expr "x$1" : 'x.*[^/]$' >/dev/null || {
+    echo "Error: URL must not end with slash"
```

Comment:
Maybe it's worth putting error messages to sterr instead of stdout? It applies to all error messages in the script.